src/fiatconnect/saga.ts

```diff
@@ -893,97 +893,120 @@ export function* handleFetchFiatConnectProviders() {
   }
 }
 
-export function* handleCreateFiatConnectTransfer({
+/**
+ * Initiates a fiatconnect transfer using the fiatconnect SDK to call a transfer/out or transfer/in endpoint
+ * for a provider.
+ **/
+export function* _initiateTransfer({
   payload: params,
 }: ReturnType<typeof createFiatConnectTransfer>) {
   const { flow, fiatConnectQuote, fiatAccountId } = params
   const quoteId = fiatConnectQuote.getQuoteId()
+  const transferType = flow === CICOFlow.CashIn ? 'transferIn' : 'transferOut'
+  Logger.info(TAG, `Starting ${transferType} ..`)
+  const fiatConnectClient: FiatConnectClient = yield call(
+    getFiatConnectClient,
+    fiatConnectQuote.getProviderId(),
+    fiatConnectQuote.getProviderBaseUrl(),
+    fiatConnectQuote.getProviderApiKey()
+  )
+  const result: Result<TransferResponse, ResponseError> = yield call(
+    [fiatConnectClient, transferType],
+    {
+      idempotencyKey: uuidv4(),
+      data: { quoteId, fiatAccountId },
+    }
+  )
+  if (result.isErr) {
+    ValoraAnalytics.track(FiatExchangeEvents.cico_fc_transfer_api_error, {
+      flow,
+      fiatConnectError: result.error.fiatConnectError,
+      error: result.error.message,
+      provider: fiatConnectQuote.getProviderId(),
+    })
+    throw result.error
+  }
+  const transferResult = result.unwrap()
 
-  if (flow === CICOFlow.CashOut) {
-    try {
-      Logger.info(TAG, 'Starting transfer out..')
-      const fiatConnectClient: FiatConnectClient = yield call(
-        getFiatConnectClient,
-        fiatConnectQuote.getProviderId(),
-        fiatConnectQuote.getProviderBaseUrl(),
-        fiatConnectQuote.getProviderApiKey()
-      )
-      const result: Result<TransferResponse, ResponseError> = yield call(
-        [fiatConnectClient, 'transferOut'],
-        {
-          idempotencyKey: uuidv4(),
-          data: { quoteId, fiatAccountId },
-        }
-      )
-
-      if (result.isErr) {
-        ValoraAnalytics.track(FiatExchangeEvents.cico_fc_transfer_api_error, {
-          flow,
-          fiatConnectError: result.error.fiatConnectError,
-          error: result.error.message,
-          provider: fiatConnectQuote.getProviderId(),
-        })
-        throw result.error
-      }
+  Logger.info(TAG, `${transferType} succeeded`, JSON.stringify(transferResult))
+  return transferResult
+}
 
-      const transferResult = result.unwrap()
+/**
+ * Initiates a transaction on-chain to send funds to a specific address. This is intended to be used for
+ * cashing out after a transfer has been initiated with a provider
+ **/
+export function* _initiateTransaction({
```

Comment:
nit: make the naming a bit more fiatconnect-specific? been digging into our methods around sending TXs recently (of which there are many!) and they all have very similar names, so good to disambiguate.